tests/test_trajectory.py

```diff
@@ -209,7 +209,7 @@ def test_load_from_ascii_file(example_model_and_crs):
             # --------- Act ----------
             # Create a trajectory from the csv file
             trajectory_from_ascii = resqpy.well.Trajectory(parent_model = model,
-                                                           deviation_survey_file = trajectory_data_file_path,
+                                                           ascii_trajectory_file = trajectory_data_file_path,
```

Comment:
We might want to set_tangent_vectors = True for one of these other tests and then check that the tangents are setup, as it will be taking a different route through the code.